applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -15,6 +15,11 @@ def __init__(self, Model, settings ):
         default_parameters = KratosMultiphysics.Parameters( """
             {
                 "model_part_name":"PLEASE_CHOOSE_MODEL_PART_NAME",
+                "angle_of_attack": 0.0,
+                "mach_infinity": 0.02941176471,
+                "density_infinity"  : 1.0,
+                "speed_of_sound": 340,
+                "heat_capacity_ratio": 1.4,
                 "mesh_id": 0,
```

Comment:
The `mesh_id` is in some processes because of legacy reasons. It is not needed anymore so you can remove it.

Reply:
Ok!